common/src/objects/state.rs

```diff
@@ -102,9 +103,9 @@ impl State {
                     MAX_CHARACTERS_WORKLOAD_NAME
                 ));
             }
-            if !re.is_match(workload_spec.agent.as_str()) {
+            if !re_agent.is_match(workload_spec.agent.as_str()) {
                 return Err(format!(
-                    "Unsupported agent name. Received '{}', expected to have characters in ^[a-zA-Z0-9_-]+[a-zA-Z0-9_-]*$",
+                    "Unsupported agent name. Received '{}', expected to have characters in ^[a-zA-Z0-9_-]*$",
```

Comment:
To avoid mistakes we should use the same constant here and at the place of creation of the regex.

Reply:
This is the same as the regex for the agent name.

Reply:
Done.